src/Symfony/Component/Console/Input/Input.php

```diff
@@ -85,6 +86,35 @@ public function getArguments(): array
         return array_merge($this->definition->getArgumentDefaults(), $this->arguments);
     }
 
+    /**
+     * Returns all the given arguments NOT merged with the default values.
+     *
+     * @param bool $strip Whether to return the raw parameters (false) or the values after the command name (true)
+     *
+     * @return array<string|bool|int|float|null|array<string|bool|int|float|null>>
+     */
+    public function getRawArguments(bool $strip = false): array
```

Comment:
I added `bool $strip = false` to keep the parity with `getRawTokens()`, but I don't really need it so I don't have an opinion as to whether it should be kept

Reply:
Just my 2 cents, but if we don't use it, it should be removed

cc @chalasr